src/pathUtils.ts

```diff
@@ -0,0 +1,47 @@
+import * as path from "path";
+
+/**
+ * Normalize a path to the standard Humanloop API format.
+ *
+ * This function is primarily used when interacting with the Humanloop API to ensure paths
```

Comment:
Why not use a standard library
https://nodejs.org/api/path.html#pathnormalizepath